### ISSUE TITLE
Change the OFF command in the climate on code 1344

### ISSUE THREAD

**Issue by Itay1787** [litinoveweedle/SmartIR#154]
Hi, need to change the shutdown command in code number 1344
Because the existing command contains an "X-FAN" function, it actually leaves the AC on ten minutes after the OFF command was given.
And this is not the desired thing by default.
So it needs to be changed

```
{
  "manufacturer": "Tadiran",
  "supportedModels": ["Remote Control YB1FA", "Tadiran Inverter"],
  "commandsEncoding": "Base64",
  "supportedController": "Broadlink",
  "temperatureUnit": "C",
  "minTemperature": 16,
  "maxTemperature": 30,
  "precision": 1,
  "operationModes": ["auto", "cool", "dry", "fan_only", "heat"],
  "fanModes": ["auto", "low", "med", "high", "turbo"],
  "swingModes": ["static", "swing", "top", "bottom"],
  "commands": {
    "off": "JgCSAAABI5QUNxQTFRMUExU2FTYVNxUSFTYVNhU3FBMUExYSFBMVEhUTFBMUFBQTFDcUNxU2FRMUExUTFBMUExU2FRMUNxQTFhIUNxUSFQACjhY2FBMUExUTFRIUExUUFBIUFBU2FRIVEhYSFTYUExUTFBQTExUTFBMUFBQTFBMUFBQTFBMVExQTFBQUExQ3FBMVAA0F",
```
**SmartIR version**
1.9.0

Thanks in advance

**Comment by litinoveweedle** [comment]
Hello, thank you for this. Will you be able to open PR for this? Or should I change it? I have no way to validate it, so your confirmation is welcome. Thank you.

**Comment by Itay1787** [comment]
> Hello, thank you for this. Will you be able to open PR for this? Or should I change it? I have no way to validate it, so your confirmation is welcome. Thank you.

Hey, sorry it took me so long to respond. I tried to open PR but had never done it before, so I didn't know what I was doing. if you can change it I will be the best.

**Comment by litinoveweedle** [comment]
Merged as #171
